frontend/src/ui/components/atoms/event-card/EventCard.scss

```diff
@@ -45,12 +47,17 @@
       display: flex;
       flex-direction: column;
       justify-content: space-evenly;
+      flex: 1;
```

Comment:
Here we have split the eventCard in two components
![Captura de pantalla 2024-05-15 a las 16 09 01](https://github.com/Inlab-Services-Rindus/rindus-app/assets/147516274/9787390f-99c2-46ff-863e-8d9876614302)



![Captura de pantalla 2024-05-15 a las 16 07 58](https://github.com/Inlab-Services-Rindus/rindus-app/assets/147516274/edfc3185-9c72-4aa8-afcd-fd16ceb2a1d0)


So my suggestion its:
-The red component should have min-width:48px
-The blue component should have width 80%